awx_collection/README.md

```diff
@@ -12,13 +12,11 @@
 [comment]: # (*  upon build of the collection                       *)
 [comment]: # (*******************************************************)
 
-This Ansible collection allows for easy interaction with an AWX server via Ansible playbooks.
+The AWX Ansible collection allows for easy interaction with an AWX server via Ansible playbooks.
```

Comment:
This also renders to become the `ansible.controller` collection on Automation Hub, so that's why this wording is written general... or templated for the specific type of build.